api/build/enqueue.go

```diff
@@ -16,28 +16,36 @@ import (
 
 // Enqueue is a helper function that pushes a queue item (build, repo, user) to the queue.
 func Enqueue(ctx context.Context, queue queue.Service, db database.Interface, item *models.Item, route string) {
-	logrus.Infof("Converting queue item to json for build %d for %s", item.Build.GetNumber(), item.Build.GetRepo().GetFullName())
+	logger := logrus.WithFields(logrus.Fields{
+		"build":    item.Build.GetNumber(),
+		"build_id": item.Build.GetID(),
+		"org":      item.Build.GetRepo().GetOrg(),
+		"repo":     item.Build.GetRepo().GetName(),
+		"repo_id":  item.Build.GetRepo().GetID(),
+	})
+
+	logger.Debug("converting queue item to json")
```

Comment:
hard to fix all of these, but maybe some specificity here

```suggestion
	logger.Debugf("enqueuing build item to route %s", route)
```

Reply:
oh, i see, this log line is a bit lower. still, feels different from how the other funcs do it. 🤷